README.md

```diff
@@ -5,158 +5,307 @@
-   
-2. Set variables at the top of the SQL script. These variables are not used to select data locations, but are 'text tags' that will be added as fields in the output file:
-    - **var_SQL_script_name**: name of the SQL script, eg `neuro_ver1p_query2_trials_2024_05_17`
-    - **var_data_trials**: `PROJ_trials_YYYYMM` Table name of clinical trials output from the Charite processing, eg `theneuro_trials_20231111`
-    - **var_data_dois**: `ORG_dois_YYYYMM` Table name containing DOIs from the partner institution, eg `theneuro_dois_20230217`
+- `P01_NAME_data_latest` - Most recent output from the BOS workflow. This is the data that the BOS dashboard draws from:
+    - `pubs` - most recent of pubsYYYYMMDD
+    - `trials` - most recent of trialsYYYYMMDD 
+    - `utility_table` - 1x1 table used for community visualizations
+    - `utility_years` - table containing year range of the data
 
-3. In the script, make sure that you are happy with the versions of the BigQuery input datasets, as these may have been updated since the last time that the script was run:
-    - **Charite Processing by the Project** - `PROJ_trials_YYYYMM` Table of output from the Charite processing, eg `theneuro_trials_20231111`
-    - **Publication DOIs** - `ORG_dois_YYYYMM` Table containing DOIs from the partner institution, eg `theneuro_dois_20230217`
-    - **Clinical Trial data extract** - This table was created in Step 1, eg `neuro_ver1p_query1_alltrials_2024_05_17`
+### Step 4 - Setup your computing environment
+**Computing**: To run the BOS workflow, a workstation (e.g. laptop computer or virtual machine) is required that either runs [Docker](https://www.docker.com/), or the BOS workflow can run a virtual environment. 
 
-5. Check that the output table has a similar naming convention to the script name, eg:
-    - `OUTPUT_ver1p_query2_trials_2024_05_17`
+**IDE**: The BOS workflow is run from within an integrated development environment (IDE) such as Visual Studio Code (i.e., [vscode](https://code.visualstudio.com/)) or [Pycharm](https://www.jetbrains.com/pycharm/). Ensure that a suitable IDE is installed.
 
-5. Make any other changes you want to make to the script and save the changes.
+**Python**: The BOS workflow is written in the [Python](https://www.python.org/) programming language. Make sure you have the correct Python version installed on your local machine. The BOS workflow is written in Python3, specifically Python 3.11. So this version of Python needs to be installed to run. If necessary then install the correct Python version. You can check that you have the correct version of python installed by typing the following command into a terminal window:
 
-6. Run the script.
+`which python3.11`
 
+**pip**: you need to have pip to install the BOS workflow. You can see if you have pip installed using the following command
```

Comment:
Depending on the mode of execution (docker or a local python env), I don't think python and pip necessarily need to be present on the system.